README.md

```diff
@@ -38,6 +38,10 @@ Let the abuse begin.
 
          make upload_server
 
+   If you only want to update wifi configuration, type:
```

Comment:
The comment in Makefile says:

> # Upload wifi configuration and init files

But here it only talks about wifi. Can you please fix the discrepancy? If this is really only wifi then change the command to `make upload_wifi_config`

Reply:
Changed the name